gradle-plugin/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.3.21'
+}
+
+dependencies {
+    implementation gradleApi()
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+}
+
+compileKotlin {
```

Comment:
Might be useful to replace with:
```
tasks.withType(org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile) {
    kotlinOptions {
        jvmTarget = "1.8"
    }
}
```

Also the snippet can be moved into the root project using `subprojects` block:
```
subprojects {
    tasks.withType(org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile) {
        kotlinOptions {
            jvmTarget = "1.8"
        }
    }
}
```